nbs/tutorial_colab.ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p> </p><p> </p><p><s>piece of the puzzle</s> step</p><p> </p><p><s>We have already setup the </s><code><s>git config</s></code><s> for our project, so all that's needed is some standard git commands to finish it off.</s> We can check in our files to our GitHub repo with the following commands:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/398/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>